Scripts/Generation_scripts/Diesel.py

```diff
@@ -18,15 +18,24 @@
 import numpy as np
 
 class Diesel():
-    def __init__(self): 
+    def __init__(self,**kwargs): 
```

Comment:
I'm including this comment so we've got a record of it:
For passing in keyword arguments, it's generally better practice to specify the kwargs explicitly, rather than unpacking from a `**kwargs` member.
Here, it would look something like:
```
    def __init__(self, location: str="", diesel_inputs: Optional[str]=None, 
```